### PR TITLE
Change standard rate for Estonia and Switzerland (valid from 2024-01-01)

### PR DESCRIPTION
**Pull request by l-naumann**
References:
[https://kpmg.com/us/en/home/insights/2023/06/tnf-switzerland-increase-of-vat-rates-effective-1-january-2024.html](https://kpmg.com/us/en/home/insights/2023/06/tnf-switzerland-increase-of-vat-rates-effective-1-january-2024.html)

[https://kpmg.com/us/en/home/insights/2023/07/tnf-estonia-direct-indirect-tax-amendments.html](https://kpmg.com/us/en/home/insights/2023/07/tnf-estonia-direct-indirect-tax-amendments.html)

### CODE REVIEW

File: vat_rates.csv [hunk 1]
```diff
@@ -316,13 +317,14 @@ UK
...
-2018-01-01,,CH,CHF,0.077,standard,Switzerland VAT on non-resident providers of digital services to local consumers.
 2017-07-01,,AU,AUD,0.1,standard,Australia VAT on non-resident providers of digital services to local consumers.
 2015-07-01,,KR,KRW,0.1,standard,South Korea VAT on non-resident providers of digital services to local consumers.
 2017-01-01,,RU,RUB,0.18,standard,Russia VAT on non-resident providers of digital services to local consumers.
 2015-10-01,,JP,JPY,0.08,standard,Japan VAT on non-resident providers of digital services to local consumers.
 2014-06-01,,ZA,ZAR,0.14,standard,South Africa VAT on non-resident providers of digital services to local consumers.
 2015-01-01,,AL,ALL,0.2,standard,Albania VAT on non-resident providers of digital services to local consumers.
 2016-10-01,,NZ,NZD,0.15,standard,New Zealand VAT on non-resident providers of digital services to local consumers.
-2011-07-01,,NO,NOK,0.25,standard,Norway VAT on non-resident providers of digital services to local consumers.
+2011-07-01,,NO,NOK,0.25,standard,Norway VAT on non-resident providers of digital services to local consumers.
```
Comment (GregOriol) [inline vat_rates.csv:330]: @l-naumann Why change this line?

**Review by GregOriol — APPROVED** [review]

File: vat_rates.csv (hunk 1 above)
Comment (l-naumann) [reply vat_rates.csv:330]: I don't quite understand that. I did not knowingly change the line and when I look at the raw version of the file, there isn't any change (also no blank line at the end).

Comment (GregOriol) [reply vat_rates.csv:330]: @l-naumann I thought it was the newline at the end too, but ok then

**Comment by kdeldycke** [comment]
Thanks @l-naumann for the update! Let's merge!

**Pull request merged**